.github/workflows/test_on_deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Run Integration Tests on Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, 3.10, 3.11]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r requirements_test.txt
+
+      - name: Run integration tests
+        run: |
+          pytest tests/integration --disable-warnings
```

Comment:
for me works only with `env PYTHONPATH=./ ` [example](https://github.com/mindsdb/minds_python_sdk/blob/e222e5df71f703eb27cf966af05a443ecd0de206/.github/workflows/test_on_pr.yml#L30)